Refresh README and fix minor issues

barchuckie opened the pull request:
I have modified README installation info. I have also fixed issues with dependencies, loading stuck  and sending empty text in the NLP view.

There was a problem with installation for some people following README guide. A few issues were received according to application run.

However, I have not included information about not recommending using this project in production and related stuff.

barchuckie committed:
Update dependencies

barchuckie committed:
Fix NLP view with empty textarea

barchuckie committed:
Add next button disablement on NLP view

barchuckie committed:
Change installing dependencies info in README

barchuckie committed:
Remove plans for the future from README

barchuckie committed:
Unite info about API credentials in README

barchuckie committed:
Correct typo in a note

TomaszGabrysiak commented on README.md:
```diff
@@ -43,6 +30,10 @@ const settings = {
+Please go to the root directory and type
+`yarn install`.
 
 ### Running the app in development mode
```
I would suggest removing these two points. Especially the production-ready build since it will be not-production ready without proxying

TomaszGabrysiak commented on src/components/nlp/view.js:
```diff
@@ -10,9 +10,12 @@ import template from './template';
-        this.updateObservations(response.mentions);
-      });
+      const feel = e.target.value;
+      if (feel) {
```
This check could be done inside of `api.parse` method so it returns an empty array in case an empty input was provided. Then you could use it without remembering this constraint.

This is not that important since API isn't a standalone library yet.

barchuckie committed:
Modify README

TomaszGabrysiak commented on README.md:
```diff
@@ -30,18 +23,45 @@ const settings = {
+
 ### Preparing a production-ready build
 
 `yarn build`
```
I would suggest removing this chapter as it might be confusing. This will be a production build from the yarn's point of view. Not the build that can be used on production according to our guidelines (eg. proxy server).

barchuckie committed:
Upgrade packages and refactor code

Upgrade most of the project dependencies.
Install Infermedica eslint configuration.
Refactor code for this new eslint config.

barchuckie committed:
Update README

Update README to version consulted with other teams.

barchuckie committed:
Change common risk factors

Modify factors in the common risk view.
Add pregnancy for >= 15 <=45 yo female and modify postmenopause to
age > 45. Add physical injury for all. Remove diabetes for age < 50.
Remove risk factor p_147.

barchuckie committed:
Add proofread README

TomaszGabrysiak commented on README.md:
````diff
@@ -34,14 +34,14 @@ const settings = {
 ```
 
-⚠️  Please do not present API credentials in public places.
+⚠️  Please do not display/use API credentials in public places/open WIFI.
````
Well, this isn't the best change. I would rephrase the whole point into: Please keep your API credentials in a secret place.

TomaszGabrysiak commented on README.md:
```diff
@@ -34,14 +34,14 @@ const settings = {
 ### Dependencies
 
-Before you start working with Infermedica API and JS ES6, please make sure that you have `node` installed in your system.
+Before you start working with Infermedica API and JS ES6, please ensure that you have `node` installed in your system.
```
I would still say: Before you start working with this example, please ... Because node isn't required for using the API in general.

barchuckie committed:
Add minor changes to README

TomaszGabrysiak reviewed (approved):
👍
